### DISCUSSION TITLE
Mystmd and Revealjs

### DISCUSSION THREAD

**Discussion by fbob** [jupyter-book/mystmd#604] [General]
Is it planned to generate [Revealjs](https://revealjs.com) presentations with mystmd ?

**Comment by agoose77** [comment]
Hi @fbob,

In terms of our near-term plan, there's no intention of starting any work on revealjs integration.

Right now, you can use something like `jupyterlab-deck` along with `jupyterlab-myst` to have MyST enabled slideshows. In the longer term, there may be an elegant way to combine MyST with reveal, e.g. using https://github.com/bouzidanas/react-reveal-slides.

I'll ping @rowanc1 in case he's given it any thought, but right now my sense is "not yet!"
